api/src/utils/mailer-templates.ts

```diff
@@ -21,132 +35,168 @@ const getAllTeams = (teamNamesToUsers: teamsUsers): string => {
                 }"'>"${
-    pitch.title
-  }"</a> has been approved! Here are the details you provided regarding the pitch:</dt>
-                <br>
-                <b>Description:</b> ${pitch.description}
+      pitch.title
+    }"</a> has been approved! Here are the details you provided regarding the pitch:</p>
                 <br>
-                <b>Link:</b> ${pitch.assignmentGoogleDocLink}
+                <b>Description:</b> <p>${pitch.description}</p>
                 <br>
                 <br>
-                <dt>Your pitch is due on ${
-                  pitch.deadline ? pitch.deadline.toDateString() : 'no date'
-                }. 
-                If you have any questions or need any additional support, please contact ${
-                  admin.email
-                }. 
-                We can't wait to see your story come together!</dt>
+                <p>
+                  Your pitch has been added to the <a href="${getPitchDocLink()}">Pitch Doc</a>
+                ${
+                  ccUser
+                    ? `
+                  Your primary editor, ${ccUser.firstName}, is cc’ed on this email and will be following up to begin discussing your story
+                  </p>
+                  <p>
+                  We can't wait to see your story come together!
+                `
+                    : ``
+                }
+                </p>
+                <p>Thanks for submitting your pitch,
                 <br>
-                <dt>Thank you,
-                    <br>
                 ${admin.preferredName || admin.firstName}
-                </dt>
+                </p>
                 </div>`,
-});
+  );
+  return { ...basicEmail, cc: ccUser ? ccUser.email : '' };
+};
 
 export const approveClaim = async (
-  author: IUser,
+  claimUser: Partial<IUser>,
   pitch: IPitchAggregate,
   admin: Partial<IUser>,
-  teams: [string],
 ): Promise<EmailMessage> => {
+  type idToTeam = Record<string, Partial<ITeam>>;
+
   const teamNamesToUsers: teamsUsers = {};
-  for (const user of pitch.assignmentContributors) {
-    for (const teamName of user.teams) {
-      const userModel = await User.findById(user.userId);
-      const info: userInfo = {
-        name: `${userModel.firstName} ${userModel.lastName}`,
-        email: userModel.email,
-      };
-      if (!(teamName in teamNamesToUsers)) {
-        teamNamesToUsers[teamName] = [info];
+  const teamIdToTeam: idToTeam = {};
+  const claimUserTeams = [];
+
+  for (const contributor of pitch.aggregated.assignmentContributors) {
+    const info: userInfo = {
+      name: `${contributor.user.firstName} ${contributor.user.lastName}`,
+      email: contributor.user.email,
+    };
+    for (const teamId of contributor.teams) {
+      const team =
+        teamId in teamIdToTeam
+          ? teamIdToTeam[teamId]
+          : await Team.findById(teamId);
+      if (!(teamId in teamIdToTeam)) {
+        teamIdToTeam[teamId] = team;
+      }
```

Comment:
Instead of creating your teams record this way, I think you can use `pitch.aggregated.teams` to construct your `teamIdtoTeam` record before your for loop at line 135 and then use that record to get information. The teams that are inside `contributor.teams` can only be ones that are already associated with a pitch so this will cover all the potential teams. This way we don't have to make extra API calls.